evergreen/staticfiles/js/functions.js

```diff
@@ -1,9 +1,18 @@
-/* Simple JavaScript to "do something/anything"
- * Automatically triggered by onload
- */
+const ws = false;
+feedPost = {}
+
 function start() {
 	document.getElementById("about").innerHTML += "<br/> We have Javascript, too! 😀"
 	document.getElementById("about").innerHTML += "<br/> This probably all gonna be replaced anyway idk"
 }
 
 
+function login() {
+	document.getElementById("loggedin").innerHTML = '<form action="/logout" method="post" enctype="application/x-www-form-urlencoded">{% csrf_token %}<input type="submit" value="Logout"></form>'
+}
+
+function logout() {
+	document.getElementById("loggedin").innerHTML = '<form action="/logout" method="post" enctype="application/x-www-form-urlencoded">{% csrf_token %}<input type="submit" value="Logout" disabled></form>'
+}
```

Comment:
This can and should be done better (completely replacing login/register with logout instead of just toggling `disabled`, saying name of user if logged in, etc), but we'll decide if that should be in-scope for this PR or not on Saturday

Reply:
This is what we decided on last meeting, but I am open to rediscuss the idea.

Reply:
Saying the name of the user I believe is required by the testing procedures. Dont remember ottomh if fully changing the login/logout is required, but if we're already doing the work to toggle logout button on and off, there's no reason not to go all the way on that